README.md

```diff
@@ -1 +1,84 @@
-Easypay Checkout SDK
+# Easypay Checkout SDK
+
+Easypay's Checkout is intended too facilitate integration with your application.
+
+## Installation:
+
+Install the package with:
+
+```sh
+npm install --save @easypaypt/checkout-sdk
+# or
+yarn add @easypaypt/checkout-sdk
+```
+
+If you do not have `yarn` installed, you can get it with `npm install --global yarn`.
+
+## Usage:
+
+### Import
+
+```js
+import { startCheckout } from "@easypaypt/checkout-sdk"
+
+const checkoutInstance = startCheckout(manifest, { id: "checkout-form" })
+```
+
+| Option     | Type     | Required | Default         | Description                                              |
+| ---------- | -------- | -------- | --------------- | -------------------------------------------------------- |
+| `manifest` | `object` | true     | `undefined`     | server to server response from checkout session creation |
+| `id`       | `string` | false    | `checkout-form` | parameter to link the element in the page                |
+
+### Linking to the Page
+
+```html
+<div id="checkout-form"></div>
+```
+
+### Hidde/Remove checkout from Page
+
+```js
+checkoutInstance.unmount()
+```
+
+### On Completion
+
+```js
+const checkoutInstance = startCheckout(manifest, {
+  onMessage: myMessageHandler,
+})
+
+function myMessageHandler(checkoutMessage) {
+  if (checkoutMessage === "complete") {
+    checkoutInstance.unmount()
+    document.write("Checkout session complete. Thank you.")
+  }
+}
+```
+
+| Option            | Type       | Required | Description                                             |
+| ----------------- | ---------- | -------- | ------------------------------------------------------- |
+| `onMessage`       | `function` | false    | callback to get notified when the user ends interaction |
+| `checkoutMessage` | `string`   | false    | parameter referencing the event that happened           |
+
```

Comment:
This is strange because they're parameters for different things.
We need to talk later about how to present this information.